docs/docs/data-sources/Website.md

```diff
@@ -0,0 +1,32 @@
+Website data sources enrich your bot's responses by connecting to available online information. If your website contains relevant content, you can utilise it to enhance your bot's answers. We have have the capability to gather data from websites that can be searched using Bing, and then scrape this data. This collected information can be used to address inquiries and provide direct links to it.
```

Comment:
`We have the capability to gather data from websites that can be searched using Bing or scrape this data.`